pkgs/development/tools/bingo/default.nix

```diff
@@ -28,12 +21,6 @@ buildGoModule rec {
 
   ldflags = [ "-s" "-w" ];
 
-  passthru.tests.version = testers.testVersion {
```

Comment:
Why did you remove the version test?

Reply:
`bingo` invokes `go` command on startup which is unavailable in version test.